src/Symfony/Component/Routing/Route.php

```diff
@@ -507,6 +514,29 @@ public function setCondition(?string $condition)
         return $this;
     }
 
+    /**
+     * Returns the priority.
+     *
+     * @return integer The priority
+     */
+    public function getPriority(): ?int
```

Comment:
As you can see in the implementation, routes don't have priorities on their own right now.
The command is not enough a reason to introduce this concept on routes IMHO.
I looked for alternatives and I propose to add a `RouteCollection::getPriorities()` method, which wouldjust return `$this->priorities`. That should allow building the command without introducing any new concepts (since "priority" is already a concept of RouteCollection)